mono/utils/mono-threads-coop.c

```diff
@@ -574,13 +573,13 @@ threads_suspend_policy_getenv (void)
 	return policy;
 }
 
-static MonoThreadsSuspendPolicy threads_suspend_policy = -1;
+static MonoThreadsSuspendPolicy threads_suspend_policy = (MonoThreadsSuspendPolicy)-1;
```

Comment:
What if we declare `static int threads_suspend_policy = -1;` and only cast to `MonoThreadsSuspendPolicy` on the return from `mono_threads_suspend_policy` once we know we definitely have a valid value?

Reply:
Yeah you could switch the types the other way around, I didn't think of that.
I still like 0 for uninitialized if you are entertaining other change. It takes no room in the executable where non-zero takes room.